internals/cli/cmd_help.go

```diff
@@ -201,7 +201,7 @@ var HelpCategories = []HelpCategory{{
 }, {
 	Label:       "Checks",
 	Description: "manage health checks",
-	Commands:    []string{"checks", "start-checks", "stop-checks", "health"},
+	Commands:    []string{"check", "checks", "start-checks", "stop-checks", "health"},
```

Comment:
I know it's simpler so it sort of makes sense for it to be first, but I think we should have `checks` first, then `check`, as the summary provided by `checks` is what most people should use first I think, and these are ordered logically (at least we attempt to).